src/SFA.DAS.EmployerCommitmentsV2.Web/Startup/AspNetStartup.cs

```diff
@@ -28,7 +36,10 @@ public void ConfigureServices(IServiceCollection services)
                 .AddDasMvc()
                 .AddUrlHelper()
                 .AddEmployerUrlHelper()
-                .AddMemoryCache();
+                .AddMemoryCache()
+                .AddDataProtection()
+                .SetApplicationName("das-employercommitments-v2-web")
+                .PersistKeysToStackExchangeRedis(redis, "DataProtection-Keys");
```

Comment:
As above, don't do this if running locally (check `_hostingEnvironment.IsDevelopment()` or similar)